exercises/css-canvas-medical/styles.css

```diff
@@ -0,0 +1,243 @@
+.container {
+  max-width: 1600px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.row-center {
+  display: flex;
+  justify-content: center;
+}
+
+.col-1-2 {
+  width: 50%;
+}
+
+.col-1-3 {
+  width: 33%;
+}
+
+.col-1-4 {
+  width: 25%;
+}
+
+.col-1-5 {
+  width: 20%;
+}
+
+a {
+  text-decoration: none;
+}
+
+.container-head {
+  border-bottom: 1px rgb(230, 230, 230) solid;
+}
+
+.container-margin {
+  width: 85%;
+  margin: auto;
+}
+
+.head-head {
+  display: flex;
+}
+
+#canvasblue {
+  height: 35px;
+  margin-top: 25px;
+  margin-bottom: 25px;
+}
+
+.row-head-link {
+  display: flex;
+}
+
+.head-link {
+  display: flex;
+  font-family: 'Avenir', sans-serif;
+  margin-top: 5%;
+  margin-right: 3%;
+  margin-left: 3%;
+  color: #a6a6a6;
+  font-size: 21px;
+}
+
+.head-contact {
+  margin-top: 2.5%;
+  border: 1px solid #294363;
+  color: white;
+  background-color: #294363;
+  font-family: 'Avenir', sans-serif;
+  border-radius: 5px;
+  font-size: 21px;
+  padding: 2% 3% 2% 3%;
+}
+
+.container-img-margin {
+  width: 85%;
+  margin: auto;
+  border-bottom: 1px lightgray solid;
+  background-image: url(images/hero.jpg);
+  background-size: cover;
+  height: 650px;
+  margin-top: 75px;
+}
+
+.img-tint {
+  background-color: rgba(0, 0, 0, 0.25);
+  width: 100%;
+  height: 100%;
+}
+
+.img-head {
+  color: white;
+  font-family: 'Tiempos Headline', serif;
+  font-size: 20px;
+  letter-spacing: 1px;
+  margin-top: 15%;
+  margin-left: 15%;
+}
+
+.img-text {
+  color: white;
+  font-family: 'Avenir', sans-serif;
+  line-height: 2;
+  font-size: 16px;
+  margin-left: 15%;
+}
+
+.img-link {
+  margin-top: 10%;
```

Comment:
These are some high margin values! Part of your problem is the use of percentages, you should keep margin and padding to `px` or `rem` wherever possible. Try that out, that may make this a little less dangerous. If you find that your values are still really high, check in with me and we can discuss if there is a better solution.

Reply:
done